kep/9-capacity-scheduling/README.md

```diff
@@ -0,0 +1,323 @@
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+<!--
+This section is incredibly important for producing high quality user-focused
+documentation such as release notes or a development roadmap.  It should be
+possible to collect this information before implementation begins in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself.  KEP editors, SIG Docs, and SIG PM
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+There is increasing demand to use Kubernetes to manage batch workloads 
+(ML/DL). In those cases, one challenge is to improve cluster utilization 
+while ensuring that each user has a reasonable amount of resources. The 
+problem can be partially addressed by the Kubernetes [ResourceQuota 
+(https://kubernetes.io/docs/concepts/policy/resource-quotas/). The native 
+Kubernetes ResourceQuota API can be used to specify the maximum overall 
+resource allocation per namespace. The quota enforcement is done through an 
+admission check. A quota consumer (e.g., a Pod) cannot be created if the 
+aggregated resource allocation exceeds the quota limit. In other words, the 
+overall resource usage is aggregated based on Pod's spec (i.e., cpu/mem 
+requests) when it's created. The Kubernetes quota design has the limitation: 
+the quota resource usage is aggregated based on the resource configurations 
+(e.g., Pod cpu/mem requests specified in the Pod spec). Although this 
+mechanism can guarantee that the actual resource consumption will never 
+exceed the ResourceQuota limit, it might lead to low resource utilization as 
+some pods may have claimed the resources but failed to be scheduled. For 
+instance, actual resource consumption may be much smaller than the limit.
+
+Due to above limitation, the batch workloads (ML/DL) can't run in a 
+Kubernetes cluster as efficiently as they do in other container orchestration
+platforms such as Yarn.In order to overcome above limitations, an 
+"ElasticQuota" concept used in [Yarn capacity scheduler](https://www
```

Comment:
This cannot be truncated. Same as above, define this link at the end of this section.

Reply:
Done